dev-python/reno/reno-3.3.0-r1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
```

Comment:
Try with `DISTUTILS_STRICT_ENTRY_POINTS=1` in make.conf.

Reply:
removed